lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -106,6 +106,9 @@
 class IngressAvailableEvent(EventBase):
     pass
 
+class IngressProxyAvailableEvent(EventBase):
```

Comment:
Shouldn't this changes be in the nginx-ingress-integrator lib?

Reply:
Yep, I've discussed with Franco that we can review the changes here first and then add them into nginx-ingress-integrator later, to avoid having two places to review changes.